dali/operators/reader/loader/loader.cc

```diff
@@ -38,6 +38,8 @@ accuracy in some cases)code", false)
       R"code(Whether accessed data should be read ahead. In case of big files like LMDB,
 RecordIO or TFRecord it will slow down first access but will decrease the time of all following
 accesses.)code", false)
+  .AddOptionalArg("target_io_bytes",
```

Comment:
I don't think we need that argument in every reader, only in the numpy one.

Reply:
fixed